app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -385,6 +391,25 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     fun clearBottomSheetState() {
         state = state.clearBottomSheetState()
     }
+    fun getOtherUserClients(): List<OtherUserClients> {
+        viewModelScope.launch {
+            otherUserClients(userId).let {
+                when (it) {
+                    is GetOtherUserClientsResult.Failure.UserNotFound -> {
+                        appLogger.e("User or Domain not found while fetching user clients ")
+                    }
+                    is GetOtherUserClientsResult.Failure.Generic -> {
+                        appLogger.e("Error while fetching the user clients : ${it.genericFailure}")
+                    }
+                    is GetOtherUserClientsResult.Success -> {
+                        state = state.copy(otherUserClients = it.otherUserClients)
+                    }
+                }
+            }
+        }
+        return state.otherUserClients
```

Comment:
Is it needed to return the list of devices, since this function will update the state anyway?

Reply:
oh yeah , I wanted to use it at first and I forgot to remove it , good catch